### ISSUE TITLE
Precompdir / wavdir behavior

### ISSUE THREAD

**Issue by slifty** [dpwe/audfprint#16]
I may be using the parameters incorrectly, but the output isn't behaving the way I would expect

I have a folder for my media `/var/audfprint/media_cache`, and folder for fingerprints `/var/audfprint/afpt_cache`

I'm passing them in using --precompdir and -wavdir as appropriate.  I would expect a fingerprint for a given media file in the media cache to appear directly in the afpt cache.  However, the resulting file is nested inside of a directory path in the afpt cache.

It would be very helpful to be able to specify that I don't want audfprint to recreate my directory structure in the fingerprint directory.

*Example Input:*

```shell
$> python audfprint.py precompute --precompdir /var/audfprint/afpt_cache --wavdir /var/audfprint/media_cache media-1.mp3
```

*Output:*

```shell
> Thu Dec 31 18:29:05 2015 precomputing hashes for /var/audfprint/media_cache/media-1.mp3 ...
> wrote /var/audfprint/afpt_cache/var/audfprint/media_cache/media-1.afpt ( 49183 hashes, 1859.936 sec)
```

*Desired Output:*

```shell
> Thu Dec 31 18:29:05 2015 precomputing hashes for /var/audfprint/media_cache/media-1.mp3 ...
> wrote /var/audfprint/afpt_cache/media-1.afpt ( 49183 hashes, 1859.936 sec)
```

**Comment by slifty** [comment]
(Note: Since this is the current behavior and changing it could break folks that have built around it, a new parameter to indicate whether the directory structure should be kept within the precompdir might be appropriate here!)

**Comment by dpwe** [comment]
Closed via 452007893d2b229c885a99f399c9be05324e22ea
